salt/modules/rpm.py

```diff
@@ -153,6 +154,77 @@ def file_list(*packages):
     return {'errors': [], 'files': ret}
 
 
+def modified(*packages, **flags):
+    """
```

Comment:
@thatch45 Yes, just yesterday I already argued that this is against Python style, and was pointed that Salt decided otherwise. Sorry I didn't know that, will fix that in different PR.